### DISCUSSION TITLE
New rapidocr_onnxruntime v1.3.9

### DISCUSSION THREAD

**Discussion by tvone** [SWHL/RapidVideOCR#49] [Q&A] [answered]
Why is it that when I try to install the rapidocr_onnxruntime v1.3.9 version, the speed is significantly slower? And does rapidocr_onnxruntime v1.2.13 support PP-OCRv4? Thank you for watching.

**Comment by tvone** [comment]
If I install version v1.3.9, can I use the improvements of PP-OCRv4? Thank you.

**Comment by SWHL** [comment, marked as answer]
`rapidocr_onnxruntime>= 1.3.x`  integrates the PP-OCR v4 model by default.

See details for [link](https://rapidai.github.io/RapidOCRDocs/docs/install_usage/rapidocr/install/)

**Comment by tvone** [comment]
ok thank you
